Gemfile

```diff
@@ -196,11 +196,16 @@ gem 'reform', '~> 1.2.6', require: false
 
 platforms :mri, :mingw, :x64_mingw do
   group :mysql2 do
-    gem 'mysql2', '~> 0.3.11'
+    # Can not be updated beyond this because of:
+    # https://github.com/brianmario/mysql2/issues/675
+    # Please also see
+    # https://github.com/rails/rails/commit/5da5e3772c32593ecf2f27b8865e81dcbe3af692
+    # meaning the limitation will be removed in rails5
+    gem 'mysql2', '~> 0.3.18'
```

Comment:
Two remarks after reading [this](https://github.com/rails/rails/issues/21544):

* apparently there is a `0.3.20` which is supported ("I downgraded the gem to 0.3.20. and everything is fine now")
* while your statement "the limitation will be removed in rails5" is technically correct, it seems that the limitation will already be removed with the next Rails 4.2 release